SunEngine/Program.cs

```diff
@@ -10,53 +9,53 @@ namespace SunEngine
 {
     public class Program
     {
-        private static StartupConfiguration StartupConfiguration;
-
-        private const string HelpCommand = "help";
-        private const string ServerCommand = "server";
-        private const string VersionCommand = "version";
-        private const string MigrateCommand = "migrate";
-        private const string InitCommand = "init";
-        private const string SeedCommand = MainSeeder.SeedCommand;
-        private const string AppendCategoriesNamesCommand = "append-cat-name";
+        private static readonly InfoPrinter InfoPrinter = new InfoPrinter();
         
         public static void Main(string[] args)
         {
-            StartupConfiguration = new StartupConfiguration(args);
+            StartupConfiguration startupConfiguration = new StartupConfiguration(args);
```

Comment:
After all changes - we can use it inside methods because it's passed through the method parameter.
It'll allow to split this class in smaller ones and write more tests and improve structure.